backupctl/version.py

```diff
@@ -3,4 +3,4 @@
 
 """Version."""
 
-__version__ = "2.0.1"
+__version__ = "2.0.2"
```

Comment:
Shouldn't this become 2.0.3 now? 2.0.2 is already released

Reply:
I thought about bumping the version __after__ i create the 2.0.3 tag, but i can change that to 2.0.3 too.

Reply:
Why would you want to do that? Then the version tagged 2.0.3 is would still report 2.0.2 in `--version`

Reply:
True, ill change it to 2.0.3